bpf/lib/common.h

```diff
@@ -589,6 +589,7 @@ enum {
 #define REASON_MISSED_CUSTOM_CALL	11
 #define REASON_DECRYPTING			12
 #define REASON_ENCRYPTING			13
+#define REASON_LB_REVNAT_DELETE 14
```

Comment:
Watch out for the alignment of values here. Tabs should be set to a size of 8 whitespaces.

Reply:
See also the comment above those defines. Any change here should be reflected on the Go side.